pkg/securelaunch/measurement/cpuid.go

```diff
@@ -0,0 +1,141 @@
+package measurement
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/intel-go/cpuid"
```

Comment:
this is a different cpuid library than we already have as a dependency through pgzip. is there any way you could use https://github.com/klauspost/cpuid instead?